toc/README.md

```diff
@@ -70,22 +55,47 @@ Retirees will become emeritus members of TOC. The list of emeritus members is as
+- The nominee must put the community first and be able to balance the relationship between the interests of the organization and the interests of the community.
 
-[TOC Meeting minutes.](https://docs.google.com/document/d/1m9Uk-JkqO_KaSZaLjL47hW1L_2Bp-yW1StCuGQzOFTk/edit)
+### Vote for election
 
+After nomination, the existing TOC members will vote for election and decide by lazy consensus.
 
 ## Decision Making
-Matters that require TOC members to make decisions can be proposed at the TOC meeting or indicated by a pull request adding an entry under the [proposal folder](./proposal).
 
-All TOC members have the right to make proposals. Only official members have the right to vote, and each has 1 vote. Decisions are made by lazy consensus. If anyone votes against it, discussion is needed. If no agreement can be reached, Majority Vote will be used.
+Within the community, different types of decisions require different forms of approval. For example, the previous section describes several decisions which require 'lazy consensus' approval. This section defines how voting is performed, the types of approvals, and which types of decision require which type of approval.
+
+### Voting
+
+Decisions regarding the community are made by votes on [the community repository](https://github.com/pingcap/community). Votes are clearly indicated by a pull request adding an entry under [votes](/votes/README.md) folder. Votes may contain multiple items for approval and these should be clearly separated. Voting is carried out by replying to the vote pull request.
+
+Voting must be open for at least 2 days. The deadline should be clearly stated in the call to vote.
+
+Voting may take three flavors:
+
+* **+1**: 'Yes,' 'Agree,' or 'the action should be performed.'
+* **0**: Neutral about the proposed action (or mildly negative but not enough so to want to block it).
+* **-1**: This is a negative vote. On issues where consensus is required, this vote counts as a veto. All vetoes must contain an explanation of why the veto is appropriate. Vetoes with no explanation are void. It may also be appropriate for a -1 vote to include an alternative course of action.
+
+All participants in the TiDB community are encouraged to show their agreement with or against a particular action by voting. But only the votes of TOC member are binding. Non-binding votes are still useful for those with binding votes to understand the perception of an action in the wider community.
+
+Only active (i.e. non-emeritus) TOC members have binding votes.
+
+### Approvals
+
+* **Lazy Consensus**: A proposal is considered supported by the community as long as nobody objects. This is the default decision-making mechanism in TiDB Community, inspired by [Apache LazyConsensus](https://www.apache.org/foundation/glossary.html#LazyConsensus). In case there is objection, and no consensus can be found, there will be a majority vote.
```

Comment:
I'm not sure, should we put a space in between `Lazy` and `Consensus`?

Reply:
Yes from the original content. Updating...

Reply:
```suggestion
* **Lazy Consensus**: A proposal is considered supported by the community as long as nobody objects. This is the default decision-making mechanism in TiDB Community, inspired by [Apache Lazy Consensus](https://www.apache.org/foundation/glossary.html#LazyConsensus). In case there is objection, and no consensus can be found, there will be a majority vote.
```